docs/guide.ipynb

```diff
@@ -0,0 +1,180 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "mechanical-pointer",
+   "metadata": {},
+   "source": [
+    "# How to customize or contribute to the knowledge base\n",
+    "\n",
+    "The [asp](https://github.com/cmudig/draco2/tree/main/draco/asp) directory contains the Draco knowledge base as answer set programs. \n",
```

Comment:
I think this should go into https://github.com/cmudig/draco2/blob/main/CONTRIBUTING.md. I imagine in the docs we should talk about how someone could add custom constraints assuming they install Draco via pypi (which we can't do yet since we haven't uploaded Draco yet). We could still link to the list of files in the asp directory from here since it's useful inspiration.

Reply:
Do you mean I should split this doc and put this part in contributing.md or just move this entire thing?

Reply:
moved the links to programs docs